platform.in

```diff
@@ -69,12 +69,20 @@
 # convert all '-' in basename to '_', add two leading underscores.
 %__brp_compress %{_rpmconfigdir}/brp-compress
 %__brp_java_gcjcompile %{_rpmconfigdir}/brp-java-bytecompile
-%__brp_python_bytecompile %{_rpmconfigdir}/brp-python-bytecompile
+%__brp_python_bytecompile %{_rpmconfigdir}/brp-python-bytecompile "%{?__python}" "%{?_python_bytecompile_errors_terminate_build}" "%{?_python_bytecompile_automagic}"
 %__brp_strip %{_rpmconfigdir}/brp-strip %{__strip}
 %__brp_strip_comment_note %{_rpmconfigdir}/brp-strip-comment-note %{__strip} %{__objdump}
 %__brp_strip_shared %{_rpmconfigdir}/brp-strip-shared
 %__brp_strip_static_archive %{_rpmconfigdir}/brp-strip-static-archive %{__strip}
 
+%disable_automagic_pybytecompile() %{expand: \
```

Comment:
I don't like this, to be honest. I think it would be better to follow `%undefine` style.

Reply:
What's `%undefine` style? Note that this does not disable the entire BRP. it changes it's behavior. How do I opt in with `%undefine`?